docs/content/en/docs-dev/user-guide/configuration-reference.md

```diff
@@ -64,6 +65,7 @@ spec:
 | timeout | duration | The maximum length of time to execute deployment before giving up. Default is 6h. | No |
 | notification | [DeploymentNotification](#deploymentnotification) | Additional configuration used while sending notification to external services. | No |
 | postSync | [PostSync](#postsync) | Additional configuration used as extra actions once the deployment is triggered. | No |
+| eventWatcher | [][EventWatcher](#eventWatcher) | Configuration for event watcher. | No |
```

Comment:
should be same as L40 and other as well 👀

Reply:
Right, thank you.